sphinxext/opengraph/__init__.py

```diff
@@ -55,19 +70,27 @@ def get_tags(
     tags += make_tag("og:description", description)
 
     # image tag
-    # Get the image from the config
+    # Get basic values from config
     image_url = config["ogp_image"]
+    ogp_use_first_image = config["ogp_use_first_image"]
+    ogp_image_alt = config["ogp_image_alt"]
+
+    if ogp_use_first_image:
+        first_image = doctree.next_node(nodes.image)
+        if Path(first_image.get("uri", "")).suffix[1:].lower() in IMAGE_MIME_TYPES:
```

Comment:
I believe this line throws an error (`'NoneType' object has no attribute 'get'`) under the following conditions:
* `ogp_use_first_image ` is set to True in conf.py.
* there is a .rst file that does not include any images.

According to README.md, the expected behaviour in a case like this is to use `ogp_image`.